jujugui/static/gui/src/app/store/env/bakery.js

```diff
@@ -515,10 +525,10 @@ YUI.add('juju-env-bakery', function(Y) {
       */
       _exportMacaroon: function (successCallback, failureCallback, response) {
         try {
-          var dm = macaroon.import(
-            JSON.parse(response.target.responseText).Macaroon
-          );
-          successCallback(dm);
+          var json = JSON.parse(response.target.responseText);
```

Comment:
this should be in a try/catch so that the GUI doesn't fall over if the server fails to send JSON